ui/src/app/job-list/table/table.component.spec.ts

```diff
@@ -279,6 +279,31 @@ describe('JobsTableComponent', () => {
     });
   }))
 
+  it('should select multiple jobs on shift-click', async( () => {
```

Comment:
Thanks for adding this, would be good to have a few more test cases to make the behavior more explicit, which should be pretty easy copy-paste-modifies of what you have e.g.:

- Shift click on an already-checked job
- Reverse order selection
- Shift click a range with an internal already-checked job in the range
- Verify behavior if you check/uncheck a job (should range select to the previous selection)

Reply:
Will do – can you clarify the last one?

Reply:
Thanks – done